server/sonar-server/src/test/java/org/sonar/server/component/NewComponentTest.java

```diff
@@ -23,14 +23,14 @@
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
-import static org.apache.commons.lang.RandomStringUtils.randomAlphabetic;
+import static com.google.common.base.Strings.repeat;
 
 public class NewComponentTest {
   @Rule
   public ExpectedException expectedException = ExpectedException.none();
 
   @Test
-  public void fail_when_key_is_null() {
+  public void fail_with_functional_exception_when_key_is_null() {
```

Comment:
name of the test is wrong, it should be something like 'fail_when_key_is_bigger_than_400_characters'